rtv/docs.py

```diff
@@ -43,3 +43,17 @@
   `LEFT` or `h`       : Return to subreddit mode
   `RIGHT` or `l`      : Fold the selected comment, or load additional comments
 """
+
+COMMENT_FILE = """
+--% {0} %--
+Do not edit the line above.
+Type your comment above the line.
+All text below the line will be discarded.
+
+If there is nothing but whitespace above the line
+then the comment will be discarded.
+
+Replying to {1}'s {2}
+
+{3}
+"""
```

Comment:
Actually git does do something similar if you use the ```-v``` flag with ```git commit```. I originally did it because I thought # was used for reddit's markdown syntax, but apparently not.

If you look at the diff that changes to using comments instead of a cutoff point, it's actually simpler to implement too.